packages/components/src/text/mixins.js

```diff
@@ -1,11 +1,8 @@
-/**
- * External dependencies
- */
-import css from '@emotion/css';
 /**
  * Internal dependencies
  */
 import { fontFamily } from './font-family';
+import css from './emotion-css';
```

Comment:
That does not seem scalable. Maybe we could export `emotion` from `@wordpress/element` or something like that?

Reply:
@gziolo do you think that `wordpress/element` will be the right place for this?

Reply:
I think the idea is to keep it as a bundled dependency of `@wordpress/components` until we decide that it scales well. @youknowriad and @ItsJonQ know better. I didn't follow that recently.

Reply:
Just found out about [@emotion/primitives](https://www.npmjs.com/package/@emotion/primitives), we should probably use that

Reply:
@Tug and I talked a bit more and because emotion is only being used on the Text component we will keep the import here.